src/pages/Account/Login.scss

```diff
@@ -6,9 +6,17 @@
     flex-flow: column;
     align-items: center;
     margin-bottom: $spacing--md;
-    padding: $spacing--lg 0;
+    padding: $spacing--lg $spacing--lg;
     box-shadow: inset 0 0 0 3px #6f43ff;
     border-radius: 0.25rem;
+    font-size: 19px;
+    transition: all 400ms $transition-function;
+    font-weight: 500;
+
+    &:hover {
+      background-color: darken($color: $primary, $amount: 20%);
+      cursor: pointer;
+    }
```

Comment:
Why are we needing to add `cursor: pointer;` here?

Reply:
So that it looks like the whole div is a button